src/coreclr/inc/corinfo.h

```diff
@@ -1735,10 +1740,6 @@ struct CORINFO_ASYNC_INFO
     CORINFO_FIELD_HANDLE continuationStateFldHnd;
     // 'Flags' field
     CORINFO_FIELD_HANDLE continuationFlagsFldHnd;
-    // 'Data' field
-    CORINFO_FIELD_HANDLE continuationDataFldHnd;
-    // 'GCData' field
-    CORINFO_FIELD_HANDLE continuationGCDataFldHnd;
     // Whether or not the continuation needs to be allocated through the
     // helper that also takes a method handle
     bool continuationsNeedMethodHandle;
```

Comment:
`bool continuationsNeedMethodHandle` is no longer needed and it can be deleted. The continuation type is attached to the collectible loader allocator now. It will prevent the loader allocator from unloading and we do not need to keep it alive explicitly.

(The shared generic handling done on the JIT side right before `continuationsNeedMethodHandle` is checked is still needed.)